packages/website/posts/2022-01-25-what-am-buying.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/87873179/146266410-dac03a35-dff7-461b-8096-254f04e5df04.jpg
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
+When you buy an NFT, you are purchasing your name at the top of the list of holders. The value comes from the previous holders (and the link to the creator), more than the image. You are also buying the right to decide who is the next token holder. You could sell it or give it away.
+
+None of this has much to do with the copyright of the linked artwork. Does a spam NFT linking to a stolen image violate copyright? What about users minting “their” version of a famous meme, and trading the NFT with friends? By setting aside concerns about the image, we can see that the power is in the link itself.
+
+NFTs are not about the scarcity of the linked image, they are a new type of link. A link that memorializes how it moves through a community, and becomes valuable because of its history. A link that gains its uniqueness from human interactions.
+
+There is enough power in this new kind of link, just because it tracks how it changed hands, to enable a market. A market of links!
+
+Where does the value come from?
+
+The authentic link to the creator and the community of buyers, is encoded in that cryptographic list of holders. Without it, the NFT loses value. In the extreme case, machines link to an unsuspecting artist’s image, and make money from uneducated buyers. As more people understand that what they’re buying is a link with the artist and community, not the image itself, the public will become more educated about what’s worth buying.
+
+Tools owe it to users to make validating the provenance of NFTs a core part of the experience. If my pfp is legendary because it used to be owned by a celebrity, I want the world to know. Those same tools will help devalue spam, and put guardrails to prevent accidentally purchasing stolen NFTs.
+
+NFT.Storage is building a dashboard to help you evaluate the health of your NFTs.
```

Comment:
Sold! ...but wait, tell me more!

Reply:
In my notes it end with "Experience the NiftyScore project here." but we need to ship that first lol